src/main/java/org/folio/rmapi/builder/VendorUrlBuilder.java

```diff
@@ -0,0 +1,68 @@
+package org.folio.rmapi.builder;
+
+import com.google.common.base.Strings;
+
+import java.io.UnsupportedEncodingException;
+import java.net.URLEncoder;
+import java.util.ArrayList;
+import java.util.List;
+
+public class VendorUrlBuilder {
+
+  private static final String VENDOR_NAME_PARAMETER = "VendorName";
+  private static final String RELEVANCE_PARAMETER = "Relevance";
```

Comment:
I understand that RM API at the moment is case in-insensitive about these parameters being passed in request but when we play around with their docs, they always have queries in lower-case. Should we stick to that convention and make these all lower-case?